### ISSUE TITLE
//ramp

### ISSUE THREAD

**Issue by VorTechnix** [sbrl/Minetest-WorldEditAdditions#36]
Building on the iterative thickness and width idea, ramp creation would be a perfect complement to 2d lines while removing the need to stack them on an axis in order to build roofs and stuff.
Suggested syntax: `//ramp <replacenode> [<incline dir>]`

**Comment by sbrl** [comment]
Oh, interesting. I assume you're thinking of a solid ramp here? This would be a great addition!

**Comment by VorTechnix** [comment]
Yup. A solid ramp. It would be nice if we could check if the replace node is a type of stair and if so make them face in the proper direction too.

**Comment by sbrl** [comment]
Cool. Ah, that could be possible - but then you'd have a solid block of stairs - I suggest a combination of `//ramp` followed by a `//overlay` in that instance might be in order (potentially followed by a `//orient`)
